lib/testing/src/lib/content-services/dialog/content-node-selector-dialog.page.ts

```diff
@@ -108,6 +108,25 @@ export class ContentNodeSelectorDialogPage {
         await this.contentList.dataTablePage().doubleClickRowByContent(name);
     }
 
+    async attachFileFromContentNode(folderName: string, fileName: string) {
+        await this.checkDialogIsDisplayed();
+        await this.checkSearchInputIsDisplayed();
+        await this.checkCancelButtonIsDisplayed();
+
+        await this.contentList.dataTablePage().waitForTableBody();
```

Comment:
you could define something like
`const dataTable = this.contentList.dataTablePage()`
and then use that variable on lines 117 -> 123

Reply:
updated